config.sh

```diff
@@ -64,8 +64,6 @@ fail() {
 	echo ""
 	echo "You need package ${pkgname} in order to compile this program."
 	echo "Please make sure it is installed."
-	echo ""
-	echo "You can download ${pkgname} from here: ${dlurl}"
```

Comment:
Line 63 defining the `dlurl` should be removed as well, as it is not used anymore.

Reply:
Fixed, please take a look.